README.md

```diff
@@ -50,7 +58,8 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+there needs to be an .append to add the users, and a definition where this code is listed.  example:$('#all_users').append(user.name);
```

Comment:
Not quite. The big problem with this code is that `UserApp.add_all_users(data);` needs to be inside of `.done()` in order to have access to the `data` argument.